lib/instance.js

```diff
@@ -22,10 +22,9 @@ var initValues = function(values, options) {
     defaults = {};
 
     if (this.Model._hasDefaultValues) {
-      Utils._.each(this.Model._defaultValues, function(valueFn, key) {
-        if (defaults[key] === undefined) {
-          defaults[key] = valueFn();
-        }
+      defaults = _.mapValues(this.Model._defaultValues, function(valueFn) {
+        var value = valueFn();
+        return value._isSequelizeMethod ? value : _.cloneDeep(value);
```

Comment:
Using that code, I get an integration test for SQLite failing, when running `npm t`:

```
  1) [SQLITE] Model create is possible to use functions as default values:
     SequelizeValidationError: string violation: year cannot be an array or an object
      at lib/instance-validator.js:72:14
  From previous event:
      at Promise.then (lib/promise.js:21:17)
      at Context.<anonymous> (test/integration/model/create.test.js:811:53)
```

It's the same error I was getting before. From my previous research, it seems to be because `valueFn()._isSequelizeMethod` isn't being correctly cloned. If this is passing on your machine, it might be a bug with some software version on mine.